query-engine/driver-adapters/connector-test-kit-executor/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "ES2022",
     "module": "ESNext",
-    "lib": ["ES2022"],
+    "lib": ["ES2022", "DOM"],
```

Comment:
what do we need the DOM for?

Reply:
`WebAssembly` global is defined only in `DOM` types for some reason.